provisioning/service.json5

```diff
@@ -0,0 +1,44 @@
+// Service configuration for production.
+{
+    "processes": {
+        "gpii": {
+            "command": "morphic-app.exe",
+            "ipc": "gpii",
+            "autoRestart": true,
+            "disabled": false,
+            env: {
+                //NODE_ENV: "app.testing"
+            }
+        }
+    },
+    "logging": {
+        "level": "DEBUG"
+    },
+    "secretFile": "%ProgramData%\\Morphic Credentials\\secret.txt",
+    "autoUpdate": {
+        "enabled": false,
+        "lastUpdatesFile": "%ProgramData%\\Morphic\\last-updates.json5",
+        "files": [{
+            // Auto-update the site config.
+            path: "%ProgramData%\\Morphic\\siteConfig.json5",
+            // Get the URL from the secrets file
+            url: "${siteConfigUrl}"
+        }],
+    },
+    "siteConfigFile": [
+        "%ProgramData%\\Morphic\\siteConfig.json5",
+        "%ProgramFiles(x86)%\\Morphic\\windows\\resources\\app\\siteConfig.json5",
+        "%ProgramFiles%\\Morphic\\windows\\resources\\app\\siteConfig.json5"
+    ],
+    "gpii-config": {
```

Comment:
better to use gpiiConfig

Reply:
Well spotted.

I did, but I wrote this file before the implementation ;)